packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -206,13 +206,10 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
         title={strings.menuTitle}
         data-ui-id={ids.participantItemMenuButton}
       >
-        <Icon
-          iconName={itemHovered ? 'ParticipantItemOptionsHovered' : 'ParticipantItemOptions'}
-          className={iconStyles}
-        />
+        <Icon iconName={'ParticipantItemOptionsHovered'} className={iconStyles} id="menuButtonIcon" />
```

Comment:
can we avoid using id? [id](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id) should be "unique across the global document" which we can never garuntee.
If we need to use `id` here use fluent's `useId()` hook to generate a more-likely-to-be unique id

Reply:
OK let me look into that! I'm using id here because using svg tag will affect the css of other icons in participant item.

Reply:
Fixed by changing id to classname :) Thanks for the suggestion!